fix: add info on operator ranking and plugs (#148)
chargetrip/developers: content/API-Reference/4+Routes/mutate-route.md

## Mutation

To get a `route ID`, you will need to provide us with EV details and at least a departure location and destination. Apart from that you can also support any of the options that are described below.

### Support alternative stations along a route

If you want to request alternative charging stations along a route, you can set the `stationsAlongRouteRadius` property of the route mutation. The radius can be set between `500` and `5000` meters. This will return all applicable stations within the set radius following the specified powers and standards.

### Support stations as waypoint

You can add a station as a waypoint or as a destination. To do this, add `stationId` to `routeRequest.via.properties` or `routeRequest.destination.properties`.

### Get additional properties

Our route destination, origin and stations have an optional `properties` attribute in any of our route queries. You can use this to fetch a complete `address`, the current temperature (`temp`) and air `pressure` at that location.

### Operator ranking

> In your route request, you can add what operators you want to prefer in order of ranking and let us know if there are operators that you wish to exclude.

Reviewer: Think this first sentence is a bit overcomplicated.

Reviewer: ```suggestion
In your route request, you can rank or exclude operators based on preference. 
```

> We support ten different levels of preference. Level 1 is considered the highest level, and level 10 is the lowest. It is possible to skip levels or add multiple operators to the same level. Every level lowers the preference by 10 percent. Prefer your ranking by setting the operator type to preferred or required to make your ranking mandatory.

Reviewer: The last sentence here is a bit hard to understand.

Reviewer: ```suggestion
We support ten different levels of preference. Level 1 is considered the highest level, and level 10 is the lowest. It is possible to skip levels or add multiple operators to the same level. Every level lowers the preference by 10 percent. To enable your operator preference make sure to set the type to `preferred`.
```
